### PR TITLE
Added check to see if configuration version path is a directory

### PR DESCRIPTION
**Pull request by elliotforbes**

### CODE REVIEW

File: configuration_version.go [hunk 1]
```diff
@@ -185,6 +186,15 @@ func (s *configurationVersions) Read(ctx context.Context, cvID string) (*Configu
 func (s *configurationVersions) Upload(ctx context.Context, url, path string) error {
 	body := bytes.NewBuffer(nil)
 
+	file, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+
+	if !file.Mode().IsDir() {
+		return errors.New("Path needs to be a directory")
```
Comment (svanharmelen) [inline configuration_version.go]: Maybe make this `Path needs to be an existing directory` to make it even more specific?

**Comment by svanharmelen** [comment]
I think you rebased against your local `master` branch, not against the `hashicorp/go-tfe` `master` branch? So there's still a merge conflict preventing to PR to get merged.

**Comment by svanharmelen** [comment]
No problem at all... Just one more thingy :wink:

**Comment by elliotforbes** [comment]
sorted 😄

**Comment by svanharmelen** [comment]
Thanks @elliotforbes! LGTM!

**Pull request merged**